src/python/dxpy/dx_extract_utils/InputsValidator.py

```diff
@@ -81,22 +85,23 @@ def interpret_conditions(self):
                 method_to_call = self.conditions_funcs.get(condition)
                 getattr(self, method_to_call)(key)
 
-    def throw_message(self, check):
+    def throw_schema_message(self, check):
         type = self.schema.get(check).get("error_message").get("type")
```

Comment:
I highly recommend renaming this variable, because `type` is a reserved keyword in Python and should not be overwritten.